<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="PICO">
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;dd&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <attribute name="title">
        <string>Configuration picoamperemetre</string>
       </attribute>
       <widget class="QFrame" name="framePICO1">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>130</y>
          <width>691</width>
          <height>381</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>196</height>
         </size>
        </property>
        <property name="palette">
         <palette>
          <active/>
          <inactive/>
          <disabled/>
         </palette>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">
border-color: rgb(0, 0, 255);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="midLineWidth">
         <number>0</number>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>3</number>
         </property>
         <item>
          <widget class="QFrame" name="frameEnergieMininum">
           <property name="palette">
            <palette>
             <active>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>240</red>
                 <green>236</green>
                 <blue>227</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>240</red>
                 <green>236</green>
                 <blue>227</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>240</red>
                 <green>236</green>
                 <blue>227</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>240</red>
                 <green>236</green>
                 <blue>227</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::WinPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Shadow::Plain</enum>
           </property>
           <property name="lineWidth">
            <number>1</number>
           </property>
           <layout class="QGridLayout" name="_12">
            <property name="spacing">
             <number>6</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="labelEnergieMinimum">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
                <bold>false</bold>
                <strikeout>false</strikeout>
                <kerning>false</kerning>
               </font>
              </property>
              <property name="text">
               <string>Energie minimum ( eV )</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLineEdit" name="lineEditEnergieMinimum">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
                <bold>true</bold>
                <strikeout>false</strikeout>
                <kerning>false</kerning>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frameEnergieMaximum">
           <property name="palette">
            <palette>
             <active>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>240</red>
                 <green>236</green>
                 <blue>227</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>240</red>
                 <green>236</green>
                 <blue>227</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>240</red>
                 <green>236</green>
                 <blue>227</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>240</red>
                 <green>236</green>
                 <blue>227</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::WinPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Shadow::Plain</enum>
           </property>
           <property name="lineWidth">
            <number>1</number>
           </property>
           <layout class="QGridLayout" name="_13">
            <property name="spacing">
             <number>6</number>
            </property>
            <item row="1" column="0">
             <widget class="QLineEdit" name="lineEditEnergieMaximum">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
                <bold>true</bold>
                <strikeout>false</strikeout>
                <kerning>false</kerning>
               </font>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="labelEnergieMaximum">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
                <bold>false</bold>
                <strikeout>false</strikeout>
                <kerning>false</kerning>
               </font>
              </property>
              <property name="text">
               <string>Energie maximum ( eV )</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="framePas">
           <property name="palette">
            <palette>
             <active>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>240</red>
                 <green>236</green>
                 <blue>227</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>240</red>
                 <green>236</green>
                 <blue>227</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>240</red>
                 <green>236</green>
                 <blue>227</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>240</red>
                 <green>236</green>
                 <blue>227</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::WinPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Shadow::Plain</enum>
           </property>
           <property name="lineWidth">
            <number>1</number>
           </property>
           <layout class="QGridLayout" name="_14">
            <property name="spacing">
             <number>6</number>
            </property>
            <item row="1" column="0">
             <widget class="QLineEdit" name="lineEditPas">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
                <bold>true</bold>
                <strikeout>false</strikeout>
                <kerning>false</kerning>
               </font>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="labelPas">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
                <bold>false</bold>
                <strikeout>false</strikeout>
                <kerning>false</kerning>
               </font>
              </property>
              <property name="mouseTracking">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Pas ( eV )</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frameDuree">
           <property name="palette">
            <palette>
             <active>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>240</red>
                 <green>236</green>
                 <blue>227</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>240</red>
                 <green>236</green>
                 <blue>227</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>240</red>
                 <green>236</green>
                 <blue>227</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>240</red>
                 <green>236</green>
                 <blue>227</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::WinPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Shadow::Plain</enum>
           </property>
           <property name="lineWidth">
            <number>1</number>
           </property>
           <layout class="QGridLayout" name="_15">
            <property name="spacing">
             <number>6</number>
            </property>
            <item row="2" column="0">
             <widget class="QLineEdit" name="lineEditDuree">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
                <bold>true</bold>
                <strikeout>false</strikeout>
                <kerning>false</kerning>
               </font>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="labelDuree">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
                <bold>false</bold>
                <strikeout>false</strikeout>
                <kerning>false</kerning>
               </font>
              </property>
              <property name="text">
               <string>Durée de mesure par énergie ( s )</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="ValidatePushButtonPICO">
        <property name="geometry">
         <rect>
          <x>-40</x>
          <y>680</y>
          <width>861</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 15pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Valider</string>
        </property>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>841</width>
          <height>87</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="HeaderPICO">
         <item>
          <widget class="QLabel" name="titrePICO">
           <property name="styleSheet">
            <string notr="true">color: rgb(170, 0, 0);
border-color: rgb(0, 0, 255);</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;CONFIGURATION PICOAMPEREMETRE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonConnexion">
           <property name="styleSheet">
            <string notr="true">font: 15pt &quot;Segoe UI&quot;;</string>
           </property>
           <property name="text">
            <string>Connexion</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="labelIntensite_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>90</y>
          <width>681</width>
          <height>35</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>false</bold>
          <strikeout>false</strikeout>
          <kerning>false</kerning>
         </font>
        </property>
        <property name="text">
         <string>Intensité :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="labelInformationBalayage">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>620</y>
          <width>671</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 16pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Prêt pour le balayage</string>
        </property>
       </widget>
      </widget>
      <widget class="QCustomPlot" name="customPlot">
       <attribute name="title">
        <string>Courbe SEY</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>869</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
